mongodb/src/main/java/org/hibernate/ogm/datastore/mongodb/MongoDBDialect.java

```diff
@@ -1636,6 +1639,84 @@ private static ReadPreference getReadPreference(AssociationContext associationCo
 		return associationContext.getAssociationTypeContext().getOptionsContext().getUnique( ReadPreferenceOption.class );
 	}
 
+
+	@Override
+	public boolean supportsNamedPosition() {
+		return false;
+	}
+
+	/**
+	 * call stored procedure
+	 * @param storedProcedureName name of stored procedure.
+	 * @param params - array with values
+	 * @param tupleContext the tuple context
+	 *@see <a href="https://stackoverflow.com/questions/32480060/call-mongodb-function-from-java"> example</a>
```

Comment:
We usually use `@see` to link to another  class or method in the project. We usually put links in the description.

In this case I would simply remove the link, I don't think it's needed.

Reply:
ok